File: Session1/Code/java/simpleproject/src/main/java/com/haufe/cookbook/SmallPizza.java
package com.haufe.cookbook;

public class SmallPizza extends Pizza {
    String owner;

    public SmallPizza(){
        this.diameter = 11;
    }

    public void SendtoCustomer(String customer){
        owner = customer;
    }

    public String GetOwner(){
        return owner;
    }

    public double PizzaArea()
    {
        return  (3.14/4) * (diameter*diameter);
    }

    public void PizzaIsOutofStock() throws Exception
    {
        throw new Exception("not allowed");
    }
}

File: Session1/Code/dotNET/console/library/SmallPizza.cs
namespace library
{
    public class SmallPizza : Pizza
    {
        string owner;

        public SmallPizza(string whatCrust) : base(whatCrust)
       {   
           this.Diameter = 11;
       }

        public string Owner
        {
            get => owner;
        }
        public void SendtoCustomer(string customer){
            owner = customer;
        }

        public override void PizzaIsOutofStock()
        {
           throw new System.Exception();
        }
    }
}